README.md

```diff
@@ -25,6 +26,24 @@ Then, you need to run below comand to install framework into your project:
 ```ruby
 pod install
 ```
+#### From Carthage
+[Carthage](https://github.com/Carthage/Carthage) is decentralized dependency manager which allows to build dependencies. 
+1. To install it, add following line to your Cartfile:
+```ruby
+github 'SwingDev/RxSwiftAutoRetry'
+```
+2. Next, run `carthage update`
+3. On your application targets’ **Build Phases** tab, in the **Link Binary With Libraries** section, drag and drop `RxAtomic.framework`, `RxSwift.framework` and `RxSwiftAutoRetry.framework` from the Carthage/Build folder on disk.
```

Comment:
Is `RxAtomic` needed ?

Reply:
RxSwift needs RxAtomic. I found on some forum that if framework A needs framework B which has dependency C, then A need both B and C.